mobile/src/navigations/main/main.tsx

```diff
@@ -1,30 +1,39 @@
-import {
-  type BottomTabNavigationOptions,
-  createBottomTabNavigator,
-} from '@react-navigation/bottom-tabs';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import React from 'react';
 
+import ChatIcon from '#assets/img/icons/chat.svg';
 import HomeIcon from '#assets/img/icons/home.svg';
 import { AppColor, MainScreenName } from '#libs/enums/enums';
 import { type TabNavigationParameterList } from '#libs/types/types';
-import { Home } from '#screens/main/home';
+import { Chat } from '#screens/chat/chat';
+import { Home } from '#screens/home/home';
 
 import { styles } from './styles';
 
 const BottomTab = createBottomTabNavigator<TabNavigationParameterList>();
 
-const tabNavigatorOptions: BottomTabNavigationOptions = {
-  tabBarActiveTintColor: AppColor.BLUE_200,
-  tabBarIcon: ({ color }) => <HomeIcon color={color} />,
-  tabBarInactiveTintColor: AppColor.GRAY_300,
+const tabNavigatorOptions = {
+  headerStyle: styles.header,
+  headerTitleStyle: styles.headerTitle,
+  tabBarActiveTintColor: AppColor.BLUE_300,
+  tabBarInactiveTintColor: AppColor.GRAY_400,
```

Comment:
Why did you change BLUE_200 to BLUE_300 and GRAY_300 to GRAY_400? It looks now like this: 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/0f333d0c-eaa6-4ee4-8973-9dc129e4e412)

I think you're just a little confused about where you need to change colors and where you don't)
This is how it looks if you use the previous colors (as well as in figma and in your screenshot in pr): 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/15a49fd8-9809-467e-9143-b13b17ef0dce)

Reply:
I have added new colors in this branch and renamed all other colours acordingly in color fils as well as in all progect, so in my branch it looks good. When I merge my branch to development it will look ok also.